Resources/Prototypes/Entities/Mobs/Species/base.yml

```diff
@@ -304,6 +304,7 @@
         Asphyxiation: -1.0
   - type: FireVisuals
     alternateState: Standing
+  - type: CharacterInformation # Parkstation-CharacterInformation
```

Comment:
This should be moved from BaseMobSpeciesOrganic to BaseMobSpecies, that way Species other than organics(such as hypothetical Cyborgs, Robot, AI, IPC characters) can have this very nice character description UI.